.envrc

```
@@ -227,6 +227,10 @@ fi
 if [ -e "$PPP_INFRA_PATH"/transcom-ppp/.envrc ]
 then
     source_env "$PPP_INFRA_PATH"/transcom-ppp/.envrc
+    behind=$(git --git-dir="$PPP_INFRA_PATH/.git" rev-list --count origin/master...master)
```

Do we want to check that `HEAD` is up to date instead of `master`? I tried to test this locally with an up to date `master` but a `HEAD` that was behind by a few commits and didn't see a warning.

That may not be a typical use case, but I think we care more about what the checked out branch is than what is sitting somewhere in git?

```suggestion
    behind=$(git --git-dir="$PPP_INFRA_PATH/.git" rev-list --count origin/master...HEAD)
```

---

I think you're right. Let me do a few more tests on this too before I move forward with it.